app/Controller/PagesController.php

```diff
@@ -36,6 +37,49 @@ class PagesController extends AppController {
  * @var array
  */
 	public $uses = array();
+	public $uses2 = array();
+	public $uses3 = array();
```

Comment:
__[development][code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Short array syntax must be used to define arrays

<!-- Comment by [SideCI](https://sideci.com) -->